templates/agenda/show.html.twig

```diff
@@ -175,15 +175,14 @@
                                 <div class="row">
                                     <div class="col">
                                         <p class="m-0 mt-2">{{ board_member.name|e }}</p>
-                                        {# TODO: Display boardmember title once it has been added to boardmember entity #}
                                         <p class="m-0" style="font-size:15px; opacity: 0.5">{{ board_member.title|e }}</p>
                                     </div>
                                     <div class="col-auto">
                                         <section class="col-md-12 mt-5 mt-md-0">
                                             <div class="row mt-2">
                                                 <div class="ml-2">
                                                     {% if not agenda.isFinished() %}
-                                                    <form id="myform" method="post" action="{{ path('agenda_board_member_remove', {'id': agenda.id, 'board_member_id': board_member.id}) }}" onsubmit="return confirm('{% trans %}Are you sure you want to remove this board member?{% endtrans %}');">
+                                                    <form method="post" action="{{ path('agenda_board_member_remove', {'id': agenda.id, 'board_member_id': board_member.id}) }}" onsubmit="return confirm('{% trans %}Are you sure you want to remove this board member?{% endtrans %}');">
```

Comment:
See the comment on how to handle the `confirm` message above.

Reply:
Good catch. This is something that will be needed in many other places, so we will fix it in a different branch.